src/adhocracy_core/adhocracy_core/sheets/principal.py

```diff
@@ -324,6 +326,48 @@ def check_plaintext_password(self, password: str) -> bool:
 )
 
 
+class ActivationSetting(SingleLine):
+    """Activation setting.
+
+    Possible values: direct, registration_mail, invitation_mail
+    """
+
+    default = 'registration_mail'
+
+    @deferred
+    def validator(self, kw: dict):
+        """Validator."""
+        return OneOf(('direct', 'registration_mail', 'invitation_mail'))
+
+    @deferred
+    def widget(self, kw: dict) -> SelectWidget:
+        choices = [(x, x) for x in self.validator.choices]
+        return SelectWidget(values=choices)
+
+
+class IActivationConfiguration(ISheet):
+    """Marker interface for the user activation configutation sheet."""
+
+
+class ActivationConfigutationSchema(MappingSchema):
+    """Data structure for user activation configuration.
+
+    `activation`: One of 'direct', 'register' or 'invite'
+    """
+
+    activation = ActivationSetting()
+
+
+activation_configuration_meta = sheet_meta._replace(
+    isheet=IActivationConfiguration,
+    schema_class=ActivationConfigutationSchema,
+    readable=False,
+    creatable=True,
```

Comment:
maybe readable is usefull, if you want to see later how the user was created/activated.